app.py

```diff
@@ -175,10 +182,17 @@ def _get_commit_sha(user, repository, commitsh):
     raise typer.Abort()
 
 
-def _side_effects(source, new_commit):
+def _side_effects(source: dict, commitsh: str):
+    """_summary_
```

Comment:
Is `_summary_` a placeholder, or does it have some special magic value?

Reply:
lol, I forgot to fill it

Reply:
it's just a placeholder